tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-literals-negative.bal

```diff
@@ -149,3 +149,8 @@ function textInvalidXmlSequence() {
     xml<'xml:Element>|'xml:Text x39 = xml `<root><foo><foo></foo>3</foo></root>3`;
     xml<xml<'xml:Text>>|xml<xml<'xml:Comment>> x40 = xml `<!--comment-->text1`;
 }
+
+function testMissingClosingGTToken() {
+    xml x1 = xml `<a /a>`;
+    xml x2 = xml `<a n="a" /a>`;
```

Comment:
@poorna2152 Shall we add this test case as a  ballerina parser test as well?

Let's add it under the  following class. https://github.com/ballerina-platform/ballerina-lang/blob/bf6b50f6b0c5dca37e978d1e0cbb9b4c4f5cfad0/compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/expressions/XMLTemplateExpressionTest.java#L27

The reason is instead of triggering the full compilation flow, we should be able to test this at the parser level alone.

Reply:
Added